docs/src/components/TailwindClassnames/index.tsx

```diff
@@ -104,6 +122,12 @@ const TailwindClassnames = () => {
             <Heading type="title2" spaceAfter="normal">
               {group}
             </Heading>
+            {group === "Colors" && (
+              <Text type="info" weight="medium" spaceAfter="medium">
+                Please remember to use default Tailwind prefixes with those classnames: bg, text,
```

Comment:
I really like that we're saying this. I would just rephrase it and also add it to other classes as well. Something like

```
To be used with Tailwind classes that apply a color property (eg: `text-blue-dark`, `bg-red-light`, etc).
```

And then for the screens something like
```
To be used as a Tailwind prefix for defining media queries (eg: `sm:opacity-1`, `tb:hidden`, etc).
```

For spacing, something like
```
To be used with Tailwind classes that apply spacing properties (eg: `p-sm`, `m-xl`, `mb-0`, etc).
```

And finally, a note on zIndex (since those are being extended, and not overwritten) saying something like
```
Default Tailwind zIndex classes are also available.
```

For all the other properties I don't think we need to add any message. Plus, the text I've suggested here is not a hard suggestion, just a content suggestion, to be coherent across all properties and informative rather than "a reminder request" (_please remember_).

Reply:
Yes, thanks for the suggestion 🚀